### PR TITLE
 Resources and credentials sections

### PR DESCRIPTION
**Pull request by vincaslt**
image

image

### CODE REVIEW

File: containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    const [addresses] = useAddresses();
+
+    const getFirstEmail = ([{ Email = '' } = {}] = []) => Email;
+    const username = isMember(user) ? getFirstEmail(addresses) : user.Name;
+
+    return (
+        <>
+            <SubTitle>{c('Title').t`ProtonVPN Credentials`}</SubTitle>
+            <Alert>
+                Use the following credentials to log into the{' '}
```
Comment (EpokK) [inline containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js]: You need to translate the string here.

File: containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import { c } from 'ttag';
+import useUser from '../../../hooks/useUser';
+import useAddresses from '../../../hooks/useAddresses';
+import { isMember } from 'proton-shared/lib/user/helpers';
+
+const ProtonVPNCredentialsSection = () => {
+    const [user] = useUser();
+    const [addresses] = useAddresses();
+
+    const getFirstEmail = ([{ Email = '' } = {}] = []) => Email;
```
Comment (EpokK) [inline containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js]: Use `Email` from `user.Email` please.

File: containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    const username = isMember(user) ? getFirstEmail(addresses) : user.Name;
+
+    return (
+        <>
+            <SubTitle>{c('Title').t`ProtonVPN Credentials`}</SubTitle>
+            <Alert>
+                Use the following credentials to log into the{' '}
+                <Href url="https://protonvpn.com/download">ProtonVPN native clients.</Href>
```
Comment (EpokK) [inline containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js]: Translation.

File: containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SubTitle, Row, Label, Field, Href } from 'react-components';
+import { c } from 'ttag';
+
+const ProtonVPNResourcesSection = () => {
+    return (
+        <>
+            <SubTitle>{c('Title').t`ProtonVPN Resources`}</SubTitle>
```
Comment (EpokK) [inline containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js]: Don't over capitalize please.
```suggestion
            <SubTitle>{c('Title').t`ProtonVPN resources`}</SubTitle>
```

File: containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            <Row>
+                <Label>{c('Label').t`Download ProtonVPN`}</Label>
+                <Field>
+                    <Href url="https://protonvpn.com/download">{c('Link').t`Download page`}</Href>
+                </Field>
+            </Row>
+            <Row>
+                <Label>{c('Label').t`ProtonVPN Homepage`}</Label>
```
Comment (EpokK) [inline containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js]:
```suggestion
                <Label>{c('Label').t`ProtonVPN homepage`}</Label>
```

File: containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+                <Field>
+                    <Href url="https://protonvpn.com/">{c('Link').t`Homepage`}</Href>
+                </Field>
+            </Row>
+            <Row>
+                <Label>{c('Label').t`Get Support for ProtonVPN`}</Label>
```
Comment (EpokK) [inline containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js]:
```suggestion
                <Label>{c('Label').t`Get support for ProtonVPN`}</Label>
```

File: containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { SubTitle, Row, Label, Field, Alert, Href } from 'react-components';
+import { c } from 'ttag';
+import useUser from '../../../hooks/useUser';
+import { isMember } from 'proton-shared/lib/user/helpers';
+
+const ProtonVPNCredentialsSection = () => {
+    const [user] = useUser();
+
+    const username = isMember(user) ? user.Email : user.Name;
```
Comment (mmso) [inline containers/vpn/ProtonVPNCredentialsSection/ProtonVPNCredentialsSection.js]:
```suggestion
    const username = user.isMember ? user.Email : user.Name;
```

**Pull request merged**